microsoft-edge/webview2/release-notes.md

```diff
@@ -117,36 +117,45 @@ For full API compatibility, this version of the WebView2 SDK requires WebView2 R
 
 ###### Bug fixes
 
-* Disabled Mouse Gesture feature by default (Runtime-only) ([Issue 3737](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3737))
-* Fixed a bug where mousewheel scrolling was intermittently broken for visual hosting (Runtime-only)
-* Fixed a bug where downloading APK files in WebView2 crashes WV2 browser process. (Runtime-only) ([Issue 3569](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3569))
+* Disabled the Mouse Gesture feature by default.  (Runtime-only)  ([Issue #3737](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3737))
+* Fixed a bug where mouse wheel scrolling was intermittently broken for visual hosting.  (Runtime-only)
+* Fixed a bug where downloading APK files in WebView2 crashes the WV2<!--todo: spell out as "WebView2"?  lowercase it?--> browser process.  (Runtime-only)  ([Issue #3569](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3569))

```

Comment:
@victorhuangwq @peiche-jessica - change "the WV2" to "the WebView2 control"?

Reply:
that works